res/controllers/Korg-KAOSS-DJ-scripts.js

```diff
@@ -0,0 +1,194 @@
+// Korg KAOSS DJ controller mapping for Mixxx
+// Seb Dooris, Fayaaz Ahmed, Lee Arromba
+
+function KAOSSDJ() {};
+                     
+const ON = 0x7F, 
```

Comment:
I am surprised that `const` works with Qt4's ancient JS interpreter. My guess is that the interpreter ignores the `const`, Since the interpreter doesn't actually support this keyword, please change this to `var` to avoid confusion.